sherpa/models/tests/test_model.py

```diff
@@ -1245,7 +1338,10 @@ def test_cache_clear_single():
     assert p._cache_ctr['misses'] == 0
 
 
-def test_cache_clear_multiple():
+@pytest.mark.parametrize('model',
+                            [Polynom1D, ExpcacheTestModel])
+
+def test_cache_clear_multiple(model):
     """Check cache_clear for a combined model."""
 
     p = Polynom1D()
```

Comment:
Did you mean this (otherwise `model` doesn't actually get used)?

```suggestion
    p = model()
```